service/awsconfig/v4/cloudconfig/cloud_config.go

```diff
@@ -42,10 +44,19 @@ func New(config Config) (*CloudConfig, error) {
 		return nil, microerror.Maskf(invalidConfigError, "logger must not be empty")
 	}
 
+	var k8sAPIExtraArgs []string
+	{
+		for arg := range config.K8sAPIExtraArgs {
+			if len(strings.SplitN(config.K8sAPIExtraArgs[arg], "=", 2)) > 0 {
```

Comment:
In this case, next parameters will pass validation:
  - `--param1` - without value
  - `--param2=val2` - with simple value
  - `-param3=arg=value` - with nested parameters

And empty flags (e.g. `--param2=`) will be skipped